packages/typedoc-plugin-appium/lib/plugin.ts

```diff
@@ -0,0 +1,27 @@
+import {Application, Context, Converter} from 'typedoc';
+import {convertCommands, createReflections} from './converter';
+import {AppiumPluginLogger} from './logger';
+import {getTheme, THEME_NAME} from './output';
+
+/**
+ * Loads the Appium TypeDoc plugin
+ * @param app - TypeDoc Application
+ */
+export function load(app: Application) {
+  const log = new AppiumPluginLogger(app.logger, 'appium');
+
+  // register our custom theme.  the user still has to choose it
+  app.renderer.defineTheme(THEME_NAME, getTheme(log));
+
+  app.converter.on(Converter.EVENT_RESOLVE_BEGIN, (ctx: Context) => {
+    // we don't want to do this work if we're not using the custom theme!
+    if (app.renderer.themeName === THEME_NAME) {
```

Comment:
can we flip the condition and turn it into a guard?

Reply:
No point, really.  We're not using `themeName` to distinguish types; we're just saying "if the enabled theme name is _our_ theme name, then do some work".  If `themeName` isn't `appium`, I don't care what happens.